nomenclature/codelist.py

```diff
@@ -153,17 +150,15 @@ def from_directory(
 
         """
         code_list, tag_dict = [], CodeList(name="tag")
-
         # parse all files in path if file is None
         file = file or "**/*"
 
-        # parse all files
-        for f in path.glob(f"{file}{ext}"):
-            with open(f, "r", encoding="utf-8") as stream:
+        for yaml_file in (f for f in path.glob(file) if f.suffix in {".yaml", ".yml"}):
```

Comment:
Maybe still refactor to simply „file“ for consistency?

Reply:
Yes that what I had originally, but `file` is already the name of the input value for the file pattern in `from_directory`. Two lines above we have `file = file or "**/*"`. I didn't want to overwrite the variable but I also did not want to change the name of the input values of `from_directory`.
If you're fine with overwriting `file`, changing the name of the input parameter or a better name than `yaml_file` (which admittedly I'm also not super happy with), I'm all up for suggestions.

Reply:
Ok, got it, then keep as is!